src/drivers/fluid_adriver.h

```diff
@@ -29,7 +29,7 @@
 
 struct _fluid_audio_driver_t
 {
-    const char *name;
+    const void *define;
```

Comment:
I would prefer `const fluid_audriver_definition_t* define;` here. Which you could do by only moving the typedef to this header while keeping the struct´s implementation opaque.